python/cudf/dataframe/dataframe.py

```diff
@@ -2344,6 +2362,35 @@ def quantile(self,
                                          quant_index=False)
         return result
 
+    def select_dtypes(self, include=None):
+        """Return a subset of the DataFrame’s columns based on the column dtypes.
+
+        Parameters
+        ----------
+        include : [type]
+            [description] (the default is None, which [default_description])
+
+        """
+
+        # TODO: mechanism to hand int->int64 float->float64...
+
+        if isinstance(include, str):
```

Comment:
What if `include` is a numpy / pandas dtype?

Reply:
That's a good question.  Last night I would looking around the codebase for unified handling of np.int64/int, float/np.float64.  Does such a thing exist ?  If not, I'll build it

Reply:
@kkraus14 what do you think about using `pd.core.dtypes.common.pandas_dtype` for checking/conversion ?

Reply:
nevermind, I'll just use `np.dtype()` should be fine

Reply:
> nevermind, I'll just use `np.dtype()` should be fine

That won't handle the `category` dtype is the only problem.

Reply:
I think you can use `pd.api.types.pandas_dtype(...)`

Reply:
Switched to `pd.api.types.pandas_dtype`